drivers/soundwire/intel.c

```diff
@@ -1589,18 +1589,21 @@ static int intel_resume_runtime(struct device *dev)
 	return _resume(dev, clock_stop);
 }
 
-#endif
-
 static const struct dev_pm_ops intel_pm = {
 	SET_SYSTEM_SLEEP_PM_OPS(intel_suspend, intel_resume)
 	SET_RUNTIME_PM_OPS(intel_suspend_runtime, intel_resume_runtime, NULL)
 };
 
+#define INTEL_PM	(&intel_pm)
+#else
+#define INTEL_PM	NULL
+#endif
+
 struct sdw_md_driver intel_sdw_driver = {
 	.driver = {
 		.name = "intel-sdw",
 		.owner = THIS_MODULE,
-		.pm = &intel_pm,
+		.pm = INTEL_PM,
```

Comment:
no, this should be done with the routines compiled out and a fallback provided.

Reply:
thansk, I will refine it.